docs/core-concepts/reusable-content-pieces/browsing-directory-of-pieces.md

```diff
@@ -43,7 +43,7 @@ modules: {
       },
       // Our new page type for displaying people
```

Comment:
Please add a comment here on a new line: "Note that the `-pages` suffix is converted to the singular `-page` for the document type.